security-policies/bundle/compliance/cis_azure/rules/cis_3_11/rule.rego

```diff
@@ -0,0 +1,31 @@
+package compliance.cis_azure.rules.cis_3_11
+
+import data.compliance.lib.common
+import data.compliance.policy.azure.data_adapter
+import future.keywords.if
+
+finding = result if {
+	# filter
+	data_adapter.is_storage_account
+	_ = data_adapter.resource.extension.blobService
```

Comment:
Why do you do this blank assignment? Is it to prevent the rule being executed if it doesn't have a blobService extension?

Reply:
Yes. If for some reason a storage account does not have the `blobService` extension it should not be evaluated for this rule (to avoid false negatives).